yarn-project/canary/scripts/docker-compose.yml

```diff
@@ -34,4 +34,5 @@ services:
       ETHEREUM_HOST: http://fork:8545
       CHAIN_ID: 31337
       PXE_URL: http://sandbox:8080
+      SEARCH_START_BLOCK: ${FORK_BLOCK_NUMBER:-0}
```

Comment:
I am unsure if this is also needed but it was used in `docker-compose-e2e-sandbox.yml` (which is used for `e2e-canary-test`), so I added it here too!

Reply:
Can't hurt